app/Http/Controllers/TagController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\TeachingObject;
+use App\Tag;
+use Illuminate\Http\Request;
+
+class TagController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+      $tags = Tag::all();
+      return view('Tag.index',['tags' => $tags]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('Tag.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+      $tag = Tag::create($request->all());
+      $tag->tos()->attach($request->input('TOs'));
```

Comment:
Esta parte creo que no es necesaria. Porque cuando tenes el ABM de Tags, no estas asignando a ningun TO.

Reply:
@ignaciojonas había pensado que tal vez un Tag estaba asignado a distintos TO, pero lo borro!